pages/members/Member.js

```diff
@@ -0,0 +1,59 @@
+import React from "react"
+import styles from "./members.module.css"
+import Image from "next/image"
+import bannerImgDefault from "../../public/blank_banner.png"
+
+export default function Member(props) {
+ 
+  const profile_image = props.profile_image_url.replace("_normal", "")
+  const description = props.description.replace(/\n/g, ' ').trim()
+
+  const [profileLoad, setProfileLoad] = React.useState(false)
+
+  function showProfile() {
+    setProfileLoad(true)
+  }
+  
+  return (
+      <div className={styles.holdBox}>
+        <a href={`https://twitter.com/${props.screen_name}`} target="_blank" className={styles.member} rel="noopener noreferrer">
+
+        {props.profile_banner_url ? //checks if member has banner set. renders default if not
+          <div>
+            <Image src={props.profile_banner_url} height="167" width="500" placeholder="blur" alt="" className={styles.banner} 
+            blurDataURL="../../blank_banner.png" 
+            unoptimized={true} // maybe only needed for development. https://github.com/vercel/next.js/issues/23590
```

Comment:
ya, I don't think you need this and that issue is marked as fixed

Reply:
as you pointed out in dm, the issue was coming from me having the config in the wrong file. fixed and `unoptimized={true}` has been removed